src/OrchardCore.Cms.Web/OrchardCore.Cms.Web.csproj

```diff
@@ -29,4 +29,4 @@
 
   <Import Project="..\OrchardCore.Build\OrchardCore.Commons.targets" />
 
-</Project>
+</Project>
```

Comment:
Please revert the uneeded changes to this file

Reply:
done